dispatchr-ruby/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,77 @@
+class UsersController < ApplicationController
+
+  def index
+    render json: User.all
+  end
+
+  def show
+    if params[:id].present?
+      user_id = params[:id]
+      user = User.find(user_id)
+
+      if user.present?
+        render json: user
+      else
+        render json: user.errors, status: :unprocessable_entity
+      end
+
+    else
+      error_str = "No/Invalid user ID entered"
+      render json: error_str, status: :unprocessable_entity
+    end
+  end
+
+  def create
+    if params[:name].present? && params[:address].present? && params[:email].present? && params[:username].present?
```

Comment:
Will it? I was under the impression that if we didn't validate the params that are inputted, whichever params that are passed are the ones that are saved and the rest stay 'nil'.

We aren't using the strong params feature.